app/src/main/java/org/sagebase/crf/step/active/HeartBeatDetector.java

```diff
@@ -150,17 +156,20 @@ void fillHsv(HeartBeatSample sample) {
         float max = Math.max(sample.r, Math.max(sample.g, sample.b));
         float delta = max - min;
 
-        if (Math.round(delta * 1000.0f) == 0) {
+        if (((int)Math.round(delta * 1000.0) == 0) || ((int)Math.round(delta * 1000.0) == 0)) {
+            sample.h = -1;
```

Comment:
Do these values have significance? Wondering what h=-1 means

Reply:
I guess -1 just means ignore the value, since "hue" can only ever be between 0 and 360 (it represents degrees), direct copy from iOS code.